includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -916,7 +938,7 @@ public function validate_upe_checkout_experience_accepted_payments_field( $key,
 	 * @return bool Whether the setting to allow saved cards is enabled or not.
 	 */
 	public function is_saved_cards_enabled() {
-		return 'yes' === $this->get_option( 'saved_cards' );
+		return $this->saved_cards;
```

Comment:
Figured we could just reference the property here now. I kept the function to avoid any unexpected changes. Plus I think it reads well and may lend itself for easier testing in the future.